fgrpc/pingsrv.go

```diff
@@ -52,9 +55,24 @@ func (s *pingSrv) Ping(c context.Context, in *PingMessage) (*PingMessage, error)
 		log.LogVf("GRPC ping: sleeping for %v", s)
 		time.Sleep(s)
 	}
+	if in.ErrorPercent > 0 && shouldInjectError(int(in.ErrorPercent)) {
+		log.Infof("GRPC ping: injecting error")
+		return nil, status.Errorf(codes.Unavailable, "Injected by fortio")
+	}
 	return &out, nil
 }
 
+func shouldInjectError(errorRate int) bool {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
this is not a good idea. rand.Seed() and Intn() are a global functions that are using mutex.
you need to create your own rand instance possibly per server instance to avoid any locking.